Automattic/wp-calypso
# Woo Installer: Landing page redesign

#### Changes proposed in this Pull Request

* Add new landing page component to /woocommerce-installation flow.
* Add new `CtaSection` component.
* Change fixed navigation component to take a `contentRef` prop and show/hide the right content depending on whether it's also visible on the page.

#### Testing instructions



* Checkout this branch and click on the "WooCommerce" nav menu item on an Atomic site that doesn't have Woo installed.
* Verify that the new landing page displays.
* Check that clicking "Set up my store!" still works to install Woo.
* Check that when you scroll past the CTA button, it appears in the header.
* Check that other pages using the fixed navigation component (like /plugins) display correctly.
* Check that you get a notice if you try to start this flow without confirming your email address.

Before
<img src="https://user-images.githubusercontent.com/811776/141930667-a17afe15-1848-43b8-ba8f-5f2750f69922.png">

After
<img src="https://user-images.githubusercontent.com/811776/141930655-a08ad8c3-d858-4204-b5be-040b55177021.png">





Related to #57419
